detox/e2e/test/channels/create_public_channel.e2e.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {toChannelScreen} from '@support/ui/screen';
+
+import {Setup} from '@support/server_api';
+
+describe('Channels', () => {
+    beforeAll(async () => {
+        const {user} = await Setup.apiInit();
+
+        await toChannelScreen(user);
+    });
+
+    it('MM-T3201 Create public channel', async () => {
+        // go to channel sidebar list
+        await element(by.id('channel_drawer_button')).tap();
+
+        // tap on + public channels
+        await element(by.id('action_button_sidebar.channels')).tap();
+
+        // expect a list of public channels, initially empty
+        await expect(element(by.text('No more channels to join'))).toBeVisible();
+
+        // tap to create new channel
+        await element(by.text('CREATE')).tap();
+
+        // expect a new screen to create a new public channel
+        await expect(element(by.text('New Public Channel'))).toBeVisible();
+
+        // fill data
+        await element(by.id('input_channel_name')).typeText('a');
+        await element(by.text('CREATE')).tap();
+
+        // the `Create` button is not tappable until at least 2 characters have been entered
```

Comment:
```suggestion
        // * Expect the `Create` button is not tappable until at least 2 characters have been entered
```

Reply:
Looks this is not yet resolved per last commit.

Reply:
right, fixed!